Yona/Yona.xcodeproj/project.pbxproj

```diff
@@ -1573,6 +1581,7 @@
 				16B1CD261D1573E400A1ABA7 /* YonaUserTableViewCell.xib in Resources */,
 				166CB8561D2FA200008B425D /* YonaButtonTableViewCell.xib in Resources */,
 				ADD108241CC78C3B0048852F /* Oswald-ExtraLight.ttf in Resources */,
+				E5B9377F210F3B2500B5413B /* fabric_script.sh in Resources */,
```

Comment:
This file shouldn't exist, so remove this line.

Reply:
We tried with the command in ```./Pods/Fabric/run "$FABRIC_API_KEY" "$FABRIC_BUILD_SECRET" ```
This doesn't seems to work directly.
Reason can be that, xcode build-phase has access to the build directory and dysm files and other needed files to the Fabric dashboard, which are not available to jenkins environment post xcodebuild step.

Reply:
Are you saying the ``fabric run` command straight from Jenkins fails?

Reply:
Yes Bert. This is error message: ``` Fabric: Failed to Detect Build Environment```

Reply:
I didn't hear about that before. In that case, make the the `fabric_script.sh` empty except for the pod run command. The environment variables should make it through to the script.